src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -208,6 +216,66 @@
  * }
  * </pre>
  * <p>
+ * To configure the check to allow methods which contain a specified comment text
+ * pattern in their javadoc to be designed for extension.
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;DesignForExtension&quot;&gt;
+ *   &lt;property name=&quot;requiredJavadocPhrase&quot;
+ *     value=&quot;This implementation&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <pre>
+ * public class A extends B {
+ *   /&#42;&#42;
+ *   &#42; This implementation ...
+ *   &#42;
+ *   &#42;/
+ *   public int foo() { // ok, required javadoc phrase in comment
+ *     return 2;
```

Comment:
please make method single line as methods below to be consistent in look.

Reply:
Done.